msteams-platform/concepts/design/design-teams-app-overview.md

```diff
@@ -134,9 +134,20 @@ Understand how people add, use, and manage Teams apps to make the most of each c
 * [Task modules](../../task-modules-and-cards/task-modules/design-teams-task-modules.md)
 * [Adaptive Cards](../../task-modules-and-cards/cards/design-effective-cards.md)
 
+## App customization
+
+Understand how the Teams admin can customize or rebrand the app based on the organization's need. This customization is enabled if you define the `configurableProperties` in the manifest schema. For more information, see [Customize apps in Microsoft Teams](/MicrosoftTeams/customize-apps).
+
+> [!NOTE]
+> The `configurableProperties` is currently available in developer preview only.
+> 
+> App customization enables admins to change the look-and-feel of the apps loaded through bots, messaging extensions, tabs, and connectors. For example, if the Teams admin customizes the name of an app from *Contoso* to *Contoso Agent*, then the app will appear with the new name *Contoso Agent* to users.
+> 
```

Comment:
@KirtiPereira suggest adding: However, while adding a connector to a chat, the list the connectors will still show the name of the app as Contoso